www/www-jekyll.yml

```diff
@@ -8,12 +8,12 @@
     tags: jekyll
 
   vars:
-    website_version: "2019.04.24"
+    website_version: "2019.05.15"
     website_name: www.openmicroscopy.org
     deploy_archive_dest_dir: /var/www/{{ website_name }}/{{ website_version }}
     deploy_archive_src_url: https://github.com/openmicroscopy/{{ website_name }}/releases/download/{{ website_version }}/{{ website_name }}.tar.gz
```

Comment:
Unrelated to this PR but I just migrated http://github.com/openmicroscopy/www.openmicroscopy.org to http://github.com/ome/www.openmicroscopy.org. The redirects are in-place but can we use this PR to also update the link and point at the new location?

Reply:
Sorry I completely missed that comment. I was focusing on ice fun!!